fabfile.py

```diff
@@ -13,7 +13,8 @@
     docker_execute_command,
     generate_satellite_docker_clients_on_rhevm,
     refresh_subscriptions_on_docker_clients,
-    remove_all_docker_containers
+    remove_all_docker_containers,
+    docker_cleanup_containers
```

Comment:
how both `remove_all_docker_containers` and `docker_cleanup_containers` are different ?